boa/examples/classes.rs

```diff
@@ -58,20 +58,20 @@ impl Class for Person {
 
     // This is what is called when we do `new Person()`
     fn constructor(_this: &Value, args: &[Value], context: &mut Context) -> Result<Self> {
-        // we get the first arguemnt of undefined if the first one is unavalable and call `to_string`.
+        // we get the first argument or undefined if the first one is unavailable and call `to_string`.
```

Comment:
```suggestion
        // we get the first argument or undefined if it is unavailable and call `to_string`.
```

Reply:
My suggestion:

We get the first argument. If it is unavailable we get `undefined`. And, then call `to_string()`.

Reply:
Sure, you can use the same sentence structure for the other comment. :+1: